modules/tinymce/src/plugins/image/main/ts/ui/Dialog.ts

```diff
@@ -326,9 +326,9 @@ const changeHandler = (helpers: Helpers, info: ImageDialogInfo, state: ImageDial
     changeFileInput(helpers, info, state, api);
   } else if (evt.name === 'isDecorative') {
     if (api.getData().isDecorative) {
-      api.disable('alt');
+      api.setEnabled('alt', false);
     } else {
-      api.enable('alt');
+      api.setEnabled('alt', true);
```

Comment:
This can be simplified significantly here to something like this (GitHub won't let me leave a suggestion sorry):
```
  api.setEnabled('alt', !api.getData().isDecorative);
```